README.md

```diff
@@ -33,55 +33,21 @@ We use bundler to manage gems. You can install bundler and other required gems l
     gem install bundler
     bundle install
 
-### Elasticsearch
+### Docker Services
 
-We're using [Elasticsearch](http://www.elasticsearch.org/) (>= 6.8) for fulltext search.
+The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.
```

Comment:
For consistency with `bot_detection`'s README and for simplicity, let's change this to:
```
### Services

The required services (MySQL, Elasticsearch, & Redis) can be run using Docker. Refer to [search-services](https://github.com/GSA/search-services) for detailed instructions.
```